### PR TITLE
Translations for the ingredients taxonomy

### PR DESCRIPTION
**Pull request by teolemon**
Translations for the ingredients taxonomy

### CODE REVIEW

**Review by stephanegigandet — APPROVED** [review]
I didn't review everything, but the sample diffs I looked at look good.

**Review by stephanegigandet — CHANGES_REQUESTED** [review]
Many tests fail because of translations of "colza oil" (huile de colza) have been added to "rapeseed oil". If there isn't a specific word for "rapeseed", then we could put "colza oil or canola oil" as the name.

e.g.

en:Rapeseed oil
fr:Huile de colza ou de navette

File: taxonomies/ingredients.txt [hunk 1]
```diff
@@ -17642,12 +18041,15 @@ sv:rypsoljeberedning
...
 ca:Oli de colza
+de:Kolzaöl, Rapsöl
 el:κραμβέλαιο
 es:aceite de colza
 fi:rapsiöljy
 fr:huile de colza, huile végétale de colza, huile colza, graisses végétales de colza, graisses vegetales de colza, colza
 it:olio di colza, olio vegetale di colza
+nb:rapsolje
 nl:koolzaadolie, plantaardige koolzaadolie
+pl:Olej rzepakowy   Linki  Wikidane  Należy:   Olej rzepakowy,
```
Comment (teolemon) [inline taxonomies/ingredients.txt:18052]: TOFIX

File: taxonomies/ingredients.txt [hunk 2]
```diff
@@ -17659,6 +18061,7 @@ nutriscore_fruits_vegetables_nuts:en:yes
 <en:colza oil
 en:organic colza oil
 de:Bio-Canola-Öl
+es:aceite de colza orgánico, aceite de colza organico,
```
Comment (teolemon) [inline taxonomies/ingredients.txt:18064]:
```suggestion
es:aceite de colza orgánico, aceite de colza organico
```

File: taxonomies/ingredients.txt (hunk 1 above)
Comment (teolemon) [reply taxonomies/ingredients.txt:18052]: fixed in commit

File: taxonomies/ingredients.txt (hunk 2 above)
Comment (teolemon) [reply taxonomies/ingredients.txt:18064]: fixed in commit

**Comment by teolemon** [comment]
- [ ] Errors in the ingredients taxonomy definition:
- [ ] ERROR - de:calciumcarbonat already is a synonym of de:e170 for entry en:e170 - de:e170i cannot be mapped to entry en:e170i
- [ ] ERROR - it:carbonato-di-calcio already is a synonym of it:carbonato-di-calcio for entry en:e170 - it:e170i cannot be mapped to entry en:e170i
- [ ] ERROR - he:לציטין already is a synonym of he:e322 for entry en:e322 - he:e322i cannot be mapped to entry en:e322i
- [ ] ERROR - he:מווסת-חומציות already is a synonym of he:מווסת-חומציות for entry en:acidity-regulator - he:e500 cannot be mapped to entry en:e500
- [ ] ERROR - he:נתרן-פחמתי already is a synonym of he:e500 for entry en:e500 - he:e500i cannot be mapped to entry en:e500i
- [ ] ERROR - de:calciumcarbonat already is a synonym of de:e170 for entry en:e170 - de:e170i cannot be mapped to entry en:e170i
- [ ] ERROR - it:carbonato-di-calcio already is a synonym of it:carbonato-di-calcio for entry en:e170 - it:e170i cannot be mapped to entry en:e170i
- [ ] ERROR - de:natriumcitrat already is a synonym of de:e331 for entry en:e331 - de:natriumcitrat cannot be mapped to entry en:sodium-citrate
- [ ] ERROR - et:naatriumtsitraat already is a synonym of et:e331 for entry en:e331 - et:naatriumtsitraat cannot be mapped to entry en:sodium-citrate
- [ ] ERROR - fi:natriumsitraatti already is a synonym of fi:e331 for entry en:e331 - fi:natriumsitraatti cannot be mapped to entry en:sodium-citrate
- [ ] ERROR - hu:nátrium-citrát already is a synonym of hu:e331 for entry en:e331 - hu:nátrium-citrát cannot be mapped to entry en:sodium-citrate
- [ ] ERROR - it:citrato-di-sodio already is a synonym of it:e331 for entry en:e331 - it:citrato-di-sodio cannot be mapped to entry en:sodium-citrate
- [ ] ERROR - sk:citran-sodný already is a synonym of sk:e331 for entry en:e331 - sk:citran-sodný cannot be mapped to entry en:sodium-citrate
- [ ] ERROR - fr:tocopherol already is a synonym of fr:e307a for entry en:e307a - fr:vitamine-e cannot be mapped to entry en:vitamin-e
- [ ] ERROR - el:dl-α-τοκοφερόλη already is a synonym of el:e307a for entry en:e307a - el:dl-α-τοκοφερόλη cannot be mapped to entry en:dl-alpha-tocopherol
- [ ] ERROR - sl:dl-alfa-tokoferol already is a synonym of sl:e307a for entry en:e307a - sl:dl-alfa-tokoferol cannot be mapped to entry en:dl-alpha-tocopherol
- [ ] ERROR - cs:sukcinát already is a synonym of cs:e363 for entry en:e363 - cs:polyethylenglykol-1000 cannot be mapped to entry en:d-alpha-tocopheryl-polyethylene-glycol-1000-succinate
- [ ] ERROR - fi:c-vitamiini already is a synonym of fi:e300 for entry en:e300 - fi:c-vitamiini cannot be mapped to entry en:vitamin-c
- [ ] ERROR - fr:vitamine-c already is a synonym of fr:e300 for entry en:e300 - fr:vitamine-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - hu:c-vitamin already is a synonym of hu:e300 for entry en:e300 - hu:c-vitamin cannot be mapped to entry en:vitamin-c
- [ ] ERROR - pt:vitamina-c already is a synonym of pt:e300 for entry en:e300 - pt:vitamina-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - sk:vitamín-c already is a synonym of sk:e300 for entry en:e300 - sk:vitamín-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - fr:palmitate-d-ascorbyle already is a synonym of fr:e304 for entry en:e304 - fr:l-ascorbyle-6-palmitate cannot be mapped to entry en:l-ascorbyl-6-palmitate
- [ ] ERROR - hu:aszkorbil-palmitát already is a synonym of hu:e304 for entry en:e304 - hu:l-aszkorbil-6-palmitát cannot be mapped to entry en:l-ascorbyl-6-palmitate
- [ ] ERROR - ca:acid-inosinic already is a synonym of ca:e630 for entry en:e630 - ca:acid-inosinic cannot be mapped to entry en:inosine-monophosphate
- [ ] ERROR - de:inosinmonophosphat already is a synonym of de:e630 for entry en:e630 - de:inosinmonophosphat cannot be mapped to entry en:inosine-monophosphate
- [ ] ERROR - es:acido-inosinico already is a synonym of es:e630 for entry en:e630 - es:acido-inosinico cannot be mapped to entry en:inosine-monophosphate
- [ ] ERROR - fi:inosiinimonofosfaatti already is a synonym of fi:e630 for entry en:e630 - fi:inosiinimonofosfaatti cannot be mapped to entry en:inosine-monophosphate
- [ ] ERROR - fr:acide-inosinique already is a synonym of fr:e630 for entry en:e630 - fr:acide-inosinique cannot be mapped to entry en:inosine-monophosphate
- [ ] ERROR - hu:inozinsav already is a synonym of hu:e630 for entry en:e630 - hu:inozinsav cannot be mapped to entry en:inosine-monophosphate
- [ ] ERROR - pt:acido-inosinico already is a synonym of pt:e630 for entry en:e630 - pt:acido-inosinico cannot be mapped to entry en:inosine-monophosphate
- [ ] ERROR - fr:chlorure-de-choline already is a synonym of fr:e1001 for entry en:e1001 - fr:chlorure-de-choline cannot be mapped to entry en:choline-chloride
- [ ] ERROR - es:caroteno already is a synonym of es:e160 for entry en:e160 - es:e160a cannot be mapped to entry en:e160a
- [ ] ERROR - es:monoestearato-de-glicerina already is a synonym of es:e471 for entry en:e471 - es:monoestearato-de-glicerina cannot be mapped to entry en:glycerol-monostearate
- [ ] ERROR - pl:monostearynian-glicerolu already is a synonym of pl:e471 for entry en:e471 - pl:monostearynian-glicerolu cannot be mapped to entry en:glycerol-monostearate
- [ ] ERROR - pl:pektyna already is a synonym of pl:e440a for entry en:e440a - pl:pektyna cannot be mapped to entry en:fruit-pectin
- [ ] ERROR - fi:natriumsakariini already is a synonym of fi:e954 for entry en:e954 - fi:natriumsakariini cannot be mapped to entry fr:e954
- [ ] ERROR - es:fermentos-lacticos-seleccionados already is a synonym of es:fermentos-lacticos for entry en:lactic-ferments - es:fermentos-lacticos-seleccionados cannot be mapped to entry en:selected-lactic-ferments
- [ ] ERROR - fr:lactique already is a synonym of fr:e270 for entry en:e270 - fr:produits-laitiers-et-derivees cannot be mapped to entry en:dairy
- [ ] ERROR - de:gerinnungsmittel already is a synonym of de:koagulat for entry en:coagulant - de:gerinnungsmittel cannot be mapped to entry en:clotting-agents
- [ ] ERROR - de:magermilchjoghurt already is a synonym of de:magermilchjoghurt for entry en:low-fat-yogurt - de:entrahmter-joghurt cannot be mapped to entry en:skimmed-yogurt
- [ ] ERROR - es:yogur-desnatado already is a synonym of es:yogur-desnatado for entry en:skimmed-yogurt - es:yogur-descremado cannot be mapped to entry en:skimmed-mild-yogurt
- [ ] ERROR - de:milchsäure already is a synonym of de:e270 for entry en:e270 - de:milchsäure cannot be mapped to entry en:acid-whey
- [ ] ERROR - de:molkepulver already is a synonym of de:molkepulver for entry en:milk-solids - de:molkenpulver cannot be mapped to entry en:whey-powder
- [ ] ERROR - es:nata-montada already is a synonym of es:nata-montada for entry en:whipping-cream - es:crema-de-leche-montada cannot be mapped to entry en:whipped-cream
- [ ] ERROR - fr:lait-caille already is a synonym of fr:lait-caille for entry en:milk-solids - fr:lait-fermente cannot be mapped to entry en:soured-milk
- [ ] ERROR - ca:formatge-fresc already is a synonym of ca:formatge-blanc-suau for entry en:soft-white-cheese - ca:formatge-fresc cannot be mapped to entry en:fromage-blanc
- [ ] ERROR - fr:retinol already is a synonym of fr:retinol for entry en:retinol - fr:vitamine-a cannot be mapped to entry en:vitamin-a
- [ ] ERROR - de:provitamin-a already is a synonym of de:e160a for entry en:e160a - de:provitamin-a cannot be mapped to entry en:provitamin-a
- [ ] ERROR - sl:tiamin-hidroklorid already is a synonym of sl:tiamin-hidroklorid for entry en:thiamin-hydrochloride - sl:tiamin cannot be mapped to entry en:thiamin
- [ ] ERROR - fr:vitamine-b8 already is a synonym of fr:biotine for entry en:biotin - fr:vitamine-b8 cannot be mapped to entry en:vitamin-b8
- [ ] ERROR - fr:folacine already is a synonym of fr:acide-folique for entry en:folic-acid - fr:folacine cannot be mapped to entry en:folacin
- [ ] ERROR - es:cianocobalamina already is a synonym of es:cianocobalamina for entry en:cyanocobalamin - es:vitamina-b12 cannot be mapped to entry en:vitamin-b12
- [ ] ERROR - pl:cyjanokobalamina already is a synonym of pl:cyjanokobalamina for entry en:cyanocobalamin - pl:witamina-b12 cannot be mapped to entry en:vitamin-b12
- [ ] ERROR - ro:cianocobalamină already is a synonym of ro:cianocobalamină for entry en:cyanocobalamin - ro:vitamina-b12 cannot be mapped to entry en:vitamin-b12
- [ ] ERROR - en:ascorbic-acid already is a synonym of en:e300 for entry en:e300 - en:vitamin-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - en:e300 already is a synonym of en:e300 for entry en:e300 - en:vitamin-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - bg:аскорбинова-киселина already is a synonym of bg:e300 for entry en:e300 - bg:витамин-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - ca:acid-ascorbic already is a synonym of ca:e300 for entry en:e300 - ca:vitamina-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - cs:kyselina-askorbová already is a synonym of cs:e300 for entry en:e300 - cs:vitamín-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - da:c-vitamin already is a synonym of da:e300 for entry en:e300 - da:c-vitamin cannot be mapped to entry en:vitamin-c
- [ ] ERROR - da:ascorbinsyre already is a synonym of da:e300 for entry en:e300 - da:c-vitamin cannot be mapped to entry en:vitamin-c
- [ ] ERROR - de:ascorbinsäure already is a synonym of de:e300 for entry en:e300 - de:vitamin-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - el:ασκορβικο-οξυ already is a synonym of el:e300 for entry en:e300 - el:βιταμίνη-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - es:acido-ascorbico already is a synonym of es:e300 for entry en:e300 - es:vitamina-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - et:askorbiinhape already is a synonym of et:e300 for entry en:e300 - et:c-vitamiin cannot be mapped to entry en:vitamin-c
- [ ] ERROR - fi:c-vitamiini already is a synonym of fi:e300 for entry en:e300 - fi:c-vitamiini cannot be mapped to entry en:vitamin-c
- [ ] ERROR - fi:askorbiinihappo already is a synonym of fi:e300 for entry en:e300 - fi:c-vitamiini cannot be mapped to entry en:vitamin-c
- [ ] ERROR - fr:vitamine-c already is a synonym of fr:e300 for entry en:e300 - fr:vitamine-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - fr:acide-ascorbique already is a synonym of fr:e300 for entry en:e300 - fr:vitamine-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - fr:acide-l-ascorbique already is a synonym of fr:e300 for entry en:e300 - fr:vitamine-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - hu:c-vitamin already is a synonym of hu:e300 for entry en:e300 - hu:c-vitamin cannot be mapped to entry en:vitamin-c
- [ ] ERROR - hu:aszkorbinsav already is a synonym of hu:e300 for entry en:e300 - hu:c-vitamin cannot be mapped to entry en:vitamin-c
- [ ] ERROR - it:acido-ascorbico already is a synonym of it:e300 for entry en:e300 - it:vitamina-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - lt:askorbo-rūgštis already is a synonym of lt:e300 for entry en:e300 - lt:vitaminas-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - lv:askorbīnskābe already is a synonym of lv:e300 for entry en:e300 - lv:c-vitamīns cannot be mapped to entry en:vitamin-c
- [ ] ERROR - mt:aċidu-askorbiku already is a synonym of mt:e300 for entry en:e300 - mt:vitamina-ċ cannot be mapped to entry en:vitamin-c
- [ ] ERROR - nl:ascorbinezuur already is a synonym of nl:e300 for entry en:e300 - nl:vitamine-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - pl:kwas-askorbinowy already is a synonym of pl:e300 for entry en:e300 - pl:witamina-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - pt:vitamina-c already is a synonym of pt:e300 for entry en:e300 - pt:vitamina-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - pt:acido-ascorbico already is a synonym of pt:e300 for entry en:e300 - pt:vitamina-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - ro:acid-ascorbic already is a synonym of ro:e300 for entry en:e300 - ro:vitamina-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - ru:аскорбиновая-кислота already is a synonym of ru:e300 for entry en:e300 - ru:аскорбиновая-кислота cannot be mapped to entry en:vitamin-c
- [ ] ERROR - sk:kyselina-askorbová already is a synonym of sk:e300 for entry en:e300 - sk:vitamín-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - sl:askorbinska-kislina already is a synonym of sl:e300 for entry en:e300 - sl:vitamin-c cannot be mapped to entry en:vitamin-c
- [ ] ERROR - sv:askorbinsyra already is a synonym of sv:e300 for entry en:e300 - sv:askorbinsyra cannot be mapped to entry en:vitamin-c
- [ ] ERROR - fr:tocopherol already is a synonym of fr:e307a for entry en:e307a - fr:vitamine-e cannot be mapped to entry en:vitamin-e
- [ ] ERROR - ro:alfa-tocoferol already is a synonym of ro:e307a for entry en:e307a - ro:alfa-tocoferol cannot be mapped to entry hr:dl-alfa-tokoferil-acetat
- [ ] ERROR - sk:vitamín-k already is a synonym of sk:vitamín-k for entry en:vitamin-k - sk:fylochinón cannot be mapped to entry en:phylloquinone
- [ ] ERROR - de:schmelzsalz already is a synonym of de:schmelzsalze for entry en:emulsifying-salts - de:schmelzsalz cannot be mapped to entry en:melting-salt
- [ ] ERROR - de:schmelzsalze already is a synonym of de:schmelzsalze for entry en:emulsifying-salts - de:schmelzsalz cannot be mapped to entry en:melting-salt
- [ ] ERROR - fr:sel-de-fonte already is a synonym of fr:sels-emulsifiants for entry en:emulsifying-salts - fr:sel-de-fonte cannot be mapped to entry en:melting-salt
- [ ] ERROR - nb:svinekjøtt already is a synonym of nb:svin for entry en:pork - nb:svinekjøtt cannot be mapped to entry en:pork-meat
- [ ] ERROR - nb:flesk already is a synonym of nb:svin for entry en:pork - nb:svinekjøtt cannot be mapped to entry en:pork-meat
- [ ] ERROR - de:schweinedarm already is a synonym of de:schweinehaut for entry en:pork-skin - de:schweinedarm cannot be mapped to entry en:pork-collagen
- [ ] ERROR - es:manteca already is a synonym of es:grasa-de-cerdo for entry en:lard - es:reduccion cannot be mapped to entry en:shortening
- [ ] ERROR - es:grasa already is a synonym of es:gordo for entry en:fat - es:reduccion cannot be mapped to entry en:shortening
- [ ] ERROR - es:oleina-de-palma already is a synonym of es:aceite-de-palma for entry en:palm-oil - es:oleina-de-palma cannot be mapped to entry en:palm-olein
- [ ] ERROR - de:rapsöl already is a synonym of de:rapsöl for entry en:rapeseed-oil - de:kolzaöl cannot be mapped to entry en:colza-oil
- [ ] ERROR - it:alcool already is a synonym of it:alcol for entry en:alcohol - it:e1510 cannot be mapped to entry en:e1510
- [ ] ERROR - de:vollkornflocken already is a synonym of de:getreidevollkornflocken for entry en:wholemeal-cereal-flakes - de:vollkornflocken cannot be mapped to entry en:whole-wheat-flakes
- [ ] ERROR - de:weizenvollkornschrot already is a synonym of de:weizenvollkornflocken for entry de:weizenvollkornflocken - de:weizenvollkornschrot cannot be mapped to entry en:whole-wheat-groats
- [ ] ERROR - sv:havregryn already is a synonym of sv:havregryn for entry en:oat-grain - sv:havreflingor cannot be mapped to entry en:oat-flakes
- [ ] ERROR - nl:volkoren-havervlokken already is a synonym of nl:volkoren-havermout for entry en:whole-grain-rolled-oats - nl:volkoren-havervlokken cannot be mapped to entry en:whole-grain-oat-flakes
- [ ] ERROR - it:fiocchi-d-avena already is a synonym of it:fiocchi-di-avena for entry en:oat-flakes - it:fiocchi-d-avena cannot be mapped to entry en:oatmeal
- [ ] ERROR - de:roggenvollkornschrot already is a synonym of de:roggenvollkornflocken for entry en:wholemeal-rye-flakes - de:roggenvollkornschrot cannot be mapped to entry en:wholemeal-rye-groats
- [ ] ERROR - de:weizenmalzmehl already is a synonym of de:weizenmalzmehl for entry en:wheat-malt-flour - de:gemaltzen-weizenmehl cannot be mapped to entry en:malted-wheat-flour
- [ ] ERROR - es:harina-de-avena-integral already is a synonym of es:harina-de-avena for entry en:oat-flour - es:harina-de-avena-integral cannot be mapped to entry en:wholemeal-oat-flour
- [ ] ERROR - fr:farine-de-caroube already is a synonym of fr:e410 for entry en:e410 - fr:caroube cannot be mapped to entry en:carob-seeds
- [ ] ERROR - fr:farine-de-graines-de-caroube already is a synonym of fr:e410 for entry en:e410 - fr:farine-de-graines-de-caroube cannot be mapped to entry en:carob-seed-flour
- [ ] ERROR - fr:farine-de-graine-de-caroube already is a synonym of fr:e410 for entry en:e410 - fr:farine-de-graines-de-caroube cannot be mapped to entry en:carob-seed-flour
- [ ] ERROR - de:guarkernmehl already is a synonym of de:e412 for entry en:e412 - de:guarkernmehl cannot be mapped to entry en:guar-flour
- [ ] ERROR - fr:farine-de-graines-de-guar already is a synonym of fr:e412 for entry en:e412 - fr:farine-de-graines-de-guar cannot be mapped to entry en:guar-flour
- [ ] ERROR - fr:farine-de-guar already is a synonym of fr:e412 for entry en:e412 - fr:farine-de-graines-de-guar cannot be mapped to entry en:guar-flour
- [ ] ERROR - fr:farine-de-graine-de-guar already is a synonym of fr:e412 for entry en:e412 - fr:farine-de-graines-de-guar cannot be mapped to entry en:guar-flour
- [ ] ERROR - nb:guarkjernemel already is a synonym of nb:e412 for entry en:e412 - nb:guarkjernemel cannot be mapped to entry en:guar-flour
- [ ] ERROR - nl:guarpitmeel already is a synonym of nl:e412 for entry en:e412 - nl:guarpitmeel cannot be mapped to entry en:guar-flour
- [ ] ERROR - sv:guarkärnmjöl already is a synonym of sv:e412 for entry en:e412 - sv:guarkärnmjöl cannot be mapped to entry en:guar-flour
- [ ] ERROR - nl:bruine-suiker already is a synonym of nl:basterdsuiker for entry en:brown-sugar - nl:bruine-suiker cannot be mapped to entry en:muscovado
- [ ] ERROR - de:vollrohrzucker already is a synonym of de:brauner-rohrzucker for entry en:brown-cane-sugar - de:vollrohrzucker cannot be mapped to entry en:whole-cane-sugar
- [ ] ERROR - nl:ruwe-rietsuiker already is a synonym of nl:ongeraffineerde-rietsuiker for entry en:unrefined-cane-sugar - nl:ruwe-rietsuiker cannot be mapped to entry en:whole-cane-sugar
- [ ] ERROR - de:mageres-kakaopulver already is a synonym of de:magerkakaopulver for entry en:skimmed-cocoa-powder - de:fettarmes-kakaopulver cannot be mapped to entry en:fat-reduced-cocoa-powder
- [ ] ERROR - de:magerkakaopulver already is a synonym of de:magerkakaopulver for entry en:skimmed-cocoa-powder - de:fettarmes-kakaopulver cannot be mapped to entry en:fat-reduced-cocoa-powder
- [ ] ERROR - it:lievito-naturale already is a synonym of it:naturale-fermento for entry en:natural-yeast - it:lievito-naturale cannot be mapped to entry en:sourdough
- [ ] ERROR - fr:phosphate already is a synonym of fr:orthophosphate for entry fr:orthophosphate - fr:phosphates cannot be mapped to entry en:phosphate
- [ ] ERROR - it:fosfato-di-magnesio already is a synonym of it:e343i for entry en:e343i - it:fosfato-di-magnesio cannot be mapped to entry en:magnesium-phosphate
- [ ] ERROR - fr:fruits-rouges already is a synonym of fr:fruits-rouges for entry en:red-fruits - fr:fruit-rouge cannot be mapped to entry en:berries
- [ ] ERROR - es:copos-de-platano already is a synonym of es:copos-de-platano for entry en:banana-flakes - es:chips-de-platano cannot be mapped to entry en:banana-chips
- [ ] ERROR - de:kandierte-kirschen already is a synonym of de:kandierte-kirsche for entry en:glace-cherry - de:kandierte-kirschen cannot be mapped to entry fr:cerises-confites
- [ ] ERROR - es:manteca-de-coco already is a synonym of es:grasa-de-coco for entry en:coconut-fat - es:manteca-de-coco cannot be mapped to entry en:coconut-butter
- [ ] ERROR - es:polvo-de-fresas already is a synonym of es:fresas-en-polvo for entry en:raspberry-powder - es:fresa-en-polvo cannot be mapped to entry en:strawberry-powder
- [ ] ERROR - de:grenache already is a synonym of de:garnacha for entry en:garnacha-grape - de:grenache cannot be mapped to entry en:grenache
- [ ] ERROR - de:gemüsemischung already is a synonym of de:gemischtes-gemüse for entry en:mixed-vegetables - de:gemüsemischung cannot be mapped to entry en:vegetable-blend
- [ ] ERROR - de:zuckerrübensirup already is a synonym of de:zuckerrübensirup for entry en:sugar-beet-syrup - de:rote-beete-sirup cannot be mapped to entry en:beetroot-syrup
- [ ] ERROR - de:kartoffelmehl already is a synonym of de:kartoffelstärke for entry en:potato-starch - de:kartoffelmehl cannot be mapped to entry en:potato-flour
- [ ] ERROR - nb:potetmel already is a synonym of nb:potetstivelse for entry en:potato-starch - nb:potetmel cannot be mapped to entry en:potato-flour
- [ ] ERROR - nl:aardappelmeel already is a synonym of nl:aardappelzetmeel for entry en:potato-starch - nl:aardappelmeel cannot be mapped to entry en:potato-flour
- [ ] ERROR - pl:mąka-ziemniaczana already is a synonym of pl:skrobia-ziemniaczana for entry en:potato-starch - pl:mąka-ziemniaczana cannot be mapped to entry en:potato-flour
- [ ] ERROR - fr:piment already is a synonym of fr:chili for entry en:chili-pepper - fr:piment cannot be mapped to entry en:pimiento
- [ ] ERROR - fr:colza already is a synonym of fr:huile-de-colza for entry en:rapeseed-oil - fr:colza cannot be mapped to entry en:rapeseed
- [ ] ERROR - fr:colza already is a synonym of fr:huile-de-colza for entry en:rapeseed-oil - fr:colza cannot be mapped to entry en:canola
- [ ] ERROR - it:piselli already is a synonym of it:piselli for entry en:green-peas - it:pisello cannot be mapped to entry en:pea
- [ ] ERROR - nl:erwten already is a synonym of nl:erwten for entry en:green-peas - nl:erwt cannot be mapped to entry en:pea
- [ ] ERROR - de:nüsse already is a synonym of de:nüssen for entry en:nut - de:nüsse cannot be mapped to entry en:tree-nut
- [ ] ERROR - fr:fruit-a-coque already is a synonym of fr:fruits-a-coque for entry en:nut - fr:fruit-a-coque cannot be mapped to entry en:tree-nut
- [ ] ERROR - nl:noten already is a synonym of nl:noot for entry en:nut - nl:noten cannot be mapped to entry en:tree-nut
- [ ] ERROR - fr:graines-de-lin already is a synonym of fr:graines-de-lin for entry en:linseed-seeds - fr:graine-de-lin cannot be mapped to entry en:flax-seed
- [ ] ERROR - he:פלפל already is a synonym of he:גמבה for entry en:bell-pepper - he:פלפל cannot be mapped to entry en:pepper
- [ ] ERROR - nb:spisskum already is a synonym of nb:spisskummen-frø for entry en:cumin-seeds - nb:spisskum cannot be mapped to entry en:caraway
- [ ] ERROR - nb:spisskummen already is a synonym of nb:spisskummen-frø for entry en:cumin-seeds - nb:spisskum cannot be mapped to entry en:caraway
- [ ] ERROR - fi:kurkuma already is a synonym of fi:e100 for entry en:e100 - fi:kurkuma cannot be mapped to entry en:turmeric
- [ ] ERROR - de:rosa-beeren already is a synonym of de:rosa-pfeffer for entry en:pink-pepper - de:rosa-beeren cannot be mapped to entry en:pink-peppercorn
- [ ] ERROR - de:süßholzwurzel already is a synonym of de:lakritz for entry en:liquorice - de:sussholzwurzel cannot be mapped to entry en:liquorice-root
- [ ] ERROR - de:hibiskus already is a synonym of de:hibiskus for entry en:hibiscus - de:karkadeblüten cannot be mapped to entry en:roselle-flower
- [ ] ERROR - fr:hibiscus already is a synonym of fr:hibiscus for entry en:hibiscus - fr:fleurs-de-carcade cannot be mapped to entry en:roselle-flower
- [ ] ERROR - fi:sahrami already is a synonym of fi:e164 for entry en:e164 - fi:sahrami cannot be mapped to entry en:saffron
- [ ] ERROR - ro:șofran already is a synonym of ro:e164 for entry en:e164 - ro:șofran cannot be mapped to entry en:saffron
- [ ] ERROR - de:grüntee already is a synonym of de:grüner-tee for entry en:green-tea - de:grüntee cannot be mapped to entry en:green-tea-leaf
- [ ] ERROR - fr:algues-euchema-transformees already is a synonym of fr:e407a for entry en:e407a - fr:algues-euchema-transformees cannot be mapped to entry en:processed-euchema-algae
- [ ] ERROR - nl:verwerkt-eucheuma-wier already is a synonym of nl:e407a for entry en:e407a - nl:verwerkt-eucheuma-wier cannot be mapped to entry en:processed-euchema-algae
- [ ] ERROR - fr:varech already is a synonym of fr:algue-marine for entry en:seaweed - fr:varech cannot be mapped to entry en:kelp
- [ ] ERROR - es:chantarela already is a synonym of es:chantarela for entry en:golden-chanterelle - es:cantharellus-cibarius cannot be mapped to entry en:cantharellus-cibarius
- [ ] ERROR - es:anchoa already is a synonym of es:anchoa for entry en:anchovy - es:boqueron cannot be mapped to entry en:european-anchovy
- [ ] ERROR - es:bonito-del-norte already is a synonym of es:atun-blanco for entry en:albacore - es:bonito-de-altura cannot be mapped to entry en:skipjack-tuna
- [ ] ERROR - ar:محار already is a synonym of ar:رخويات for entry en:mollusc - ar:محار cannot be mapped to entry en:oyster
- [ ] ERROR - es:pota already is a synonym of es:calamar for entry en:squid - es:calamar-de-humboldt cannot be mapped to entry en:humboldt-squid
- [ ] ERROR - ca:gall-dindi already is a synonym of ca:gall-dindi for entry en:turkey - ca:carn-de-gall-dindi cannot be mapped to entry en:turkey-meat
- [ ] ERROR - de:biskuit already is a synonym of de:kekse for entry en:biscuit - de:biskuitkuchen cannot be mapped to entry en:sponge-cake
- [ ] ERROR - de:gemüsebrühe already is a synonym of de:gemüsebrühe for entry en:vegetable-shortening - de:gemüsebouillon cannot be mapped to entry en:vegetable-broth
- [ ] ERROR - fr:melon-amer already is a synonym of fr:melon-amer for entry en:bitter-melon - fr:momordique cannot be mapped to entry en:momordic

**Comment by teolemon** [comment]
@stephanegigandet I somehow fixed it, but there are many non blocking errors that are disturbing.

File: taxonomies/ingredients.txt [hunk 3]
```diff
@@ -17958,19 +17958,16 @@ wikipedia:en:https://en.wikipedia.org/wiki/Olive_pomace_oil
...
 <en:vegetable oil and fat
 en:rapeseed oil, rapeseed fat, rapeseed oil and fat
-ca:Oli de colza
 cs:řepkový olej, olej řepkový
 da:rapsolie
 de:Rapsöl, Rapsol
 es:aceite de nabina, aceite vegetal de nabina
 et:rapsiõli, rapsiseemneõli
 fi:rapsiöljy, rapsirasva, rapsiöljy ja -rasva
-fr:Huile de colza
```
Comment (stephanegigandet) [inline taxonomies/ingredients.txt]: we should add "fr:Huile de colza ou de navette" as the French name of en:rapeseed oil

**Review by stephanegigandet — APPROVED** [review]
I guess we can merge as-is, and then we can review and correct the non blocking errors one by one.

**Comment by teolemon** [comment]
I guess so. Collecting feedback is important though.

**Comment by teolemon** [comment]
@stephanegigandet fixed

**Comment by teolemon** [comment]
Just rebased. Let's merge immediately once it passes @stephanegigandet

**Pull request merged**